core/src/com/unciv/logic/city/CityReligion.kt

```diff
@@ -214,4 +215,24 @@ class CityInfoReligionManager {
         
         updateNumberOfFollowers()
     }
+    
+    /** Doesn't update the pressures, only returns what they are if the update were to happen right now */
+    fun getPressuresFromSurroundingCities(): Counter<String> {
+        val addedPressure = Counter<String>()
+        if (cityInfo.isHolyCity()) {
+            addedPressure[religionThisIsTheHolyCityOf!!] = 5 * pressureFromAdjacentCities
+        }
+        val allCitiesWithin10Tiles =
+            cityInfo.civInfo.gameInfo.getCities()
+                .filter {
+                    it != cityInfo
+                            && it.getCenterTile().aerialDistanceTo(cityInfo.getCenterTile()) <= 10
+                }
+        for (city in allCitiesWithin10Tiles) {
+            val majorityReligionOfCity = city.religion.getMajorityReligion() ?: continue
+            if (!cityInfo.civInfo.gameInfo.religions[majorityReligionOfCity]!!.isMajorReligion()) continue
```

Comment:
I think there should be a getMajorityReligionObject() function in this class, seems like something that has many uses